tests/e2e/pageObjects/pages/Transactions/Cancel.ts

```diff
@@ -0,0 +1,77 @@
+import { by, element } from 'detox';
+
+import actions from '../../../actions';
+import MessageScreen from '../../common/MessageScreen';
+import TransactionConfirmationScreen from '../../common/TransactionConfirmationScreen';
+import TransactionPasswordScreen from '../../common/TransactionPasswordScreen';
+
+const CancelTransaction = () => {
+  const TransactionListScreen = () => ({
+    nextButton: element(by.id('cancel-transaction-next-button')),
+
+    async chooseTransactionFromTheList(note: string) {
+      await actions.tap(element(by.id(`transaction-item-${note}-checkbox`)));
+    },
+
+    async tapNextButton() {
+      await actions.tap(this.nextButton);
+    },
+  });
+
+  const TransactionDetailsScreen = () => ({
+    walletAddressInput: element(by.id('cancel-transaction-wallet-address-input')),
+    useWalletAddressButton: element(by.id('use-address-of-this-wallet-item')),
+    nextButton: element(by.id('cancel-transaction-wallet-details-next-button')),
+
+    async typeWalletAddress(address: string) {
+      await actions.typeText(this.walletAddressInput, address);
+    },
+
+    async tapUseWalletAddressButton() {
+      await actions.tap(this.useWalletAddressButton);
+    },
+
+    async tapNextButton() {
+      await actions.tap(this.nextButton);
+    },
+  });
+
+  const SeedPhraseScreen = () => ({
+    self: 'cancel-seed-phrase-screen',
+    cancelButton: element(by.id('cancel-seed-phrase-cancel-button')),
+
+    async typePrivateCancelKeySeedPhrase(seedPhrases: string) {
+      await this.typeSeedPhraseFrom(seedPhrases.split(' '));
+    },
+
+    async typePrivateFastKeySeedPhrase(seedPhrases: string) {
+      await this.typeSeedPhraseFrom(seedPhrases.split(' '));
+    },
+
+    async tapCancelButton() {
+      await actions.tap(this.cancelButton);
+    },
+
+    //Private function
+    //Seed Phrase list index starts from 0 but index for id starts from 1
```

Comment:
Use different comment style `/**  */` to describe the method and the note `// Seed Phrase list index starts from 0 but index for id starts from 1` should be next to the line that it's about

Reply:
done